core/unit_test/tstNeighborList.hpp

```diff
@@ -786,31 +796,33 @@ void testFirstNeighborParallelReduce()
     auto positions = Cabana::slice<0>( aosoa );
 
     // Create the neighbor list.
-    using ListType =
-        Cabana::VerletList<TEST_MEMSPACE, Cabana::FullNeighborTag, LayoutTag>;
+    using ListType = Cabana::VerletList<TEST_MEMSPACE, Cabana::FullNeighborTag,
+                                        LayoutTag, Cabana::TeamOpTag>;
     double grid_min[3] = {box_min, box_min, box_min};
     double grid_max[3] = {box_max, box_max, box_max};
     ListType nlist( positions, 0, aosoa.size(), test_radius, cell_size_ratio,
                     grid_min, grid_max );
 
-    // Reduction variables
-    double test_sum = 0;
-    double serial_sum = 0;
-    double team_sum = 0;
-
     // Test the list parallel operation by adding a value from each neighbor
     // to the particle and compare to counts.
     auto sum_op = KOKKOS_LAMBDA( const int i, const int n, double &sum )
     {
         sum += positions( i, 0 ) + positions( n, 0 );
     };
     Kokkos::RangePolicy<TEST_EXECSPACE> policy( 0, aosoa.size() );
+
+    // Do the reductions.
+    double test_sum = 0;
```

Comment:
I would move the definition closer to its point of use in the block starting line 828